src/replication.c

```diff
@@ -590,6 +596,29 @@ trigger_rebuild(spec_t *spec)
 	if (dw_replica == NULL)
 		return;
 
+#ifdef DEBUG
+	const char* non_zero_inflight_replica_str =
+	    getenv("non_zero_inflight_replica_cnt");
+	unsigned int non_zero_inflight_replica_cnt = 0;
+	io_queue_time[0].tv_sec = 0;
+
+	if (non_zero_inflight_replica_str != NULL)
+		non_zero_inflight_replica_cnt =
+	            (unsigned int)strtol(non_zero_inflight_replica_str, NULL, 10);
+	if (non_zero_inflight_replica_cnt == 1)
+		io_queue_time[0].tv_sec = dw_replica->create_time.tv_sec - 1;
+#endif
+
+	/*
+	 * Check if healthy replica or all replica have any inflight IOs
```

Comment:
comment can be `check if any inflight IOs queued before dw_replica opened data connection`

Reply:
ok